src/main/java/core/basesyntax/strategy/impl/SupplyOperationImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class SupplyOperationImpl implements OperationHandler {
+    @Override
+    public void applyOperation(FruitTransaction transaction) {
+        if (transaction.getQuantity() <= MIN_QUANTITY) {
```

Comment:
The `MIN_QUANTITY` constant is used here but it has not been declared anywhere within this class or imported. Ensure that you have defined this constant in an appropriate place and that it is accessible within this class.